test/index.test.ts

```diff
@@ -0,0 +1,10 @@
+// You can import your modules
+// import index from '../lib/index'
```

Comment:
Potentially could do `'../src/index'` instead if you wanted to reference the TypeScript source rather than the compiled JavaScript... there are pros and cons for each approach.